testing/py-xattr/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Adam Ruzicka <a.ruzicka@outlook.com>
+pkgname=py-xattr
+_pkgname=pyxattr
+pkgver=0.6.0
+pkgrel=0
+pkgdesc="A python module for manipulating extended attributes"
+url="https://pyxattr.k1024.org/"
+arch="all"
+license="GPL"
+depends="python2"
+depends_dev=""
+makedepends="python2-dev attr-dev"
+install=""
+subpackages=""
+source="https://pyxattr.k1024.org/downloads/pyxattr-$pkgver.tar.gz"
+
+_builddir="$srcdir"/$_pkgname-$pkgver
```

Comment:
No need custom _builddir here. You just could override $builddir and use everywhere when need it.